pkg/adapter/registrar.go

```diff
@@ -30,6 +30,9 @@ type Registrar interface {
 
 	// RegisterQuota registers a new Quota builder.
 	RegisterQuota(QuotaBuilder)
+
+	// RegisterMetrics registers a new Metrics factory.
```

Comment:
factory -> builder. Let's keep the names consistent until/if we refactor again...